compute_tools/src/compute.rs

```diff
@@ -316,6 +345,475 @@ pub(crate) fn construct_superuser_query(spec: &ComputeSpec) -> String {
 }
 
 impl ComputeNode {
+    #[allow(clippy::too_many_arguments)]
+    pub fn new(
```

Comment:
Maybe this would be better as `impl From<Cli> for ComputeNode`?

Reply:
Looking at the usage, this actually sounds as a good suggestion

Reply:
I think we are hitting this problem where the `compute_tools` crate is needlessly decoupled from `compute_ctl`. `compute_ctl` should probably just be a top-level crate with most of `compute_tools` folded in.

In any case, I'm not sure the params struct is much better, but it could be short-lived if others agree with my previous paragraph

Reply:
yeah, some restructuring like that would be in order...

Now that we have another binary, `fast_import` in the same crate, the situation is this:

compute_ctl:
- compute_tools/src/bin/compute_ctl.rs
- compute_tools/src/*.rs
- compute_tools/src/http/
- libs/vm_monitor/

fast_import:
- compute_tools/src/bin/fast_import.rs
- compute_tools/src/bin/fast_import/*.rs

I feel it should be more like this:

compute_ctl:
- compute/compute_ctl/src/main.rs
- compute/compute_ctl/src/*.rs
- compute/compute_ctl/src/vm_monitor/
- compute/compute_ctl/src/http/

fast_import:
- compute/fast_import/src/*.rs

I won't try to do that in this PR though.